src/RegistryCoordinator.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity ^0.8.12;
 
 import {IPauserRegistry} from "eigenlayer-contracts/src/contracts/interfaces/IPauserRegistry.sol";
 import {ISignatureUtils} from "eigenlayer-contracts/src/contracts/interfaces/ISignatureUtils.sol";
+import {AVSDirectory} from "eigenlayer-contracts/src/contracts/core/AVSDirectory.sol";
```

Comment:
Nit: can we import interface? Does the interface not have all functions we need?

Reply:
It was there now [b69d253](https://github.com/Layr-Labs/eigenlayer-middleware/pull/287/commits/b69d253bdf9bff683fdd3e21410b0cb9f6ca7f14)